README.md

```diff
@@ -29,3 +39,39 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+# Tests
+ensure pytest is installed
+>>> pip install pytest pytest-flask
+to runn all tests, navigate to the project folder and, from the command line, run the below
+>>> pytest
+
+
```

Comment:
You could also add a line saying that there is the option of running the app in a VM with Vagrant.